### ISSUE TITLE
class-methods-use-this and React componentWillUnmount using es6

### ISSUE THREAD

**Issue by cristian-sima** [eslint/eslint#7411]
This code 

```jsx
  componentWillUnmount () {
    setOriginalPageTitle();
  }
```
gives me:

![image](https://cloud.githubusercontent.com/assets/12199042/19561050/bd5e6f20-96df-11e6-99a1-0b3475936dae.png)

which is actually perfect (because after my React component is unmounted, I clear the page title, but I do not need anything about the class).

**Comment by eslintbot** [comment]
Hi @cristian-sima, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by vitorbal** [comment]
@cristian-sima I think you're looking for the `exceptMethods` option. Take a look at an example usage fine-tuned for React here: https://github.com/eslint/eslint/issues/7085#issuecomment-250465391

**Comment by cristian-sima** [comment]
Yes. Tx
